portal-impl/src/resource-actions/documentlibrary.xml

```diff
@@ -19,6 +19,7 @@
 				<action-key>ADD_REPOSITORY</action-key>
 				<action-key>ADD_SHORTCUT</action-key>
 				<action-key>ADD_STRUCTURE</action-key>
+				<action-key>ADVANCE_UPDATE</action-key>
```

Comment:
do we need this? I see that this ADVANCE_UPDATE permission is also added to Documents, see this image:
![image](https://github.com/user-attachments/assets/e26f0f84-c07d-4fba-a5d5-8843ab315402)

Reply:
yes, for some reason it also expects and advance_permission for the model @Mikellorza

Reply:
Ok, that's fine then